dist/setup-appliance.sh

```diff
@@ -586,8 +587,15 @@ if [[ ! $BOOTSTRAP_TEST_MODE == 1 && $0 != "-bash" ]];then
   # prepare configuration for obssigner before any other backend service
   # is started, because obssigner configuration might affect other services
   # too
+  GPG_KEY_CREATED=0
+
   prepare_obssigner
 
+  if [[ $GPG_KEY_CREATED == 1 ]];then
+    /usr/lib/obs/server/bs_srcserver --restart
+    /usr/lib/obs/server/bs_repserver --restart
```

Comment:
can you use the init script and use "reload" here instead?

It would not start them when they are not running.